### ISSUE TITLE
Version of Cascadia Mono?

### ISSUE THREAD

**Issue by tiffany352** [ryanoasis/nerd-fonts#520]
#### 🎯 Subject of the issue

Cascadia Mono is a version of Cascadia Code that doesn't have ligatures. I'd like if Nerd Fonts supported it.

Currently, the Windows Terminal does not support _disabling_ ligatures. As a result, there's no way for me to turn off ligatures without either not using Nerd Fonts, or to choose a Nerd Fonts distribution for some other font.

#### 🔧 Your Setup

- _Which font are you using (e.g. `Anonymice Powerline Nerd Font Complete.ttf`)?_ Caskaydia Cove Nerd Font, v2.1.0. I tried all 4 variations in the .zip, they all have ligatures. I thought the two that say "Mono" wouldn't, but they do.
- _Which terminal emulator are you using (e.g. `iterm2`, `urxvt`, `gnome`, `konsole`)?_ Windows Terminal 1.2.2381.0
- _Are you using OS X, Linux or Windows? And which specific version or distribution?_ Windows 10

#### ★ Optional

I'm trying Nerd Fonts because I'm using Starship as my shell's prompt.

Edit: It looks like Cascadia Mono PL covers the symbols that Starship uses either way, so it seems like I don't need this.

**Comment by ryanoasis** [comment]
thanks for the suggestion. glad you can get what you need for now with Cascadia Mono PL

**Comment by trallnag** [comment]
Took me a bit of time to figure out the difference between the two meanings of "Mono". For Cascadia it means no ligatures, for Nerd Fonts "Mono" is something completely different

**Comment by ffes** [comment]
I fell for the same trap, thinking that "Caskaydia Cove Nerd Mono" was the font without the ligatures.

It turns out since Windows Terminal v1.11 it is possible to disable the ligatures. See the docs at https://docs.microsoft.com/en-us/windows/terminal/customize-settings/profile-appearance#font-features

Note that turning the ligatures off by adding just `"liga": 0` as a font feature doesn't work for "Caskaydia Cove Nerd" or "Caskaydia Cove Nerd Mono". After digging some more I found https://github.com/microsoft/terminal/issues/759#issuecomment-931743104 With disabling a lot more features, you can turn off the ligatures. A bit of a hassle, but it works.

So having a Caskadya variant without the ligatures would be much more handy. I would suggest to call it "Caskadya Mono Nerd", to keep the naming it in line with the naming convention of the original font. But I don't mind how it is called, as long as it is clear what the differences are between them.

For convenience I pasted the full `features` object I added. I haven't investigated which one of these features actually does the trick. Note that you need to edit `settings.json` for this.
 
```json
"features": {
    "calt": 0,
    "clig": 0,
    "dlig": 0,
    "hlig": 0,
    "liga": 0,
    "rlig": 0,
    "salt": 1
}
```

**Comment by Finii** [comment]
I believe there are a lot of people out there that rely on having ligatures in the Nerd Font Mono; me for one ;)

Well, I would change the source to `Cascadia Mono` and create a full set of
1. Caskaydia Mono Nerd Font
1. Caskaydia Mono Nerd Font Mono

Interestingly the RFN is already in place.

Having two different meanings for `Mono` is widespread for Nerd Fonts, for example
* Blex Mono Nerd Font Mono
* Ubuntu Mono Nerd Font Mono
* Fura Mono Nerd Font Mono

As for 'what mono means', yes the docu can and should be improved. Even more so if we add another third variant.
* `Nerd Font` (non-mono): Not monospaced, Symbols added ((\*) scaled down to max 2 'widths')
* `Nerd Font Mono`: Unify advance width for all glyphs, added symbols scaled down to fit into one width
* `Nerd Font MonoBigIcons`: Unify advance width for all glyphs, added symbols may extend to up to 2 'widths'

Removal of ligatures does not happen in `font-patcher` but only in `gotta-patch-em-all`, for some fonts, like ... manually, inconsistent with what people that self-patch usually get.

(\*) Not in effect at the moment, #748

**Comment by Finii** [comment]
> I haven't investigated which one of these features actually does the trick.

Well, tables with `lig` in the name usually concern ligatures, while `alt` are for alternative characters.

**Comment by Finii** [comment]
Closing because transferred to https://github.com/ryanoasis/nerd-fonts/discussions/1095

**Comment by DimDroll** [comment]
I came to this discussion while searching for solution in google. Earlier comment confused me a little, so I removed:
```
salt
calt
```
From the list of the features, so I still had characters like:
`--------------`
Replaced with an alternatives.

Make sure to use feature set that @ffes provided.

Complete feature list can be found here:
https://learn.microsoft.com/zh-cn/typography/opentype/spec/featurelist
Thanks to the author of the following post for pointing it out here:
https://github.com/microsoft/terminal/issues/12213#issuecomment-1019648108
